pypif_sdk/func/calculate_funcs.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*- 
+from __future__ import unicode_literals
+import pypif
+import re
+from pypif.obj import *
+import json
+import os.path
+
+path = os.path.abspath(os.path.dirname(__file__))
+path_to_json = os.path.join(path, "elements.json")
```

Comment:
This minimally needs to be added to the manifest to get deployed.  I suggest using `pkg_resources`, though.